src/components/account/details/UpdateToggle.js

```diff
@@ -1,13 +1,46 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { toggleEdit } from '../../../actions/account_details'
+import Button from './Button'
 
 import '../../../styles/account/details/update_toggle.css'
 
-export default class UpdateToggle extends Component {
+class UpdateToggle extends Component {
+  handleClick() {
+    this.props.toggleEdit()
+  }
+
   render() {
-    return (
-      <div className="edit-details__button">
-        <span>{ this.props.name }</span>
-      </div>
-    )
+    if ( !this.props.editing ) {
+      return (
+        <Button styleName="edit-details__button"
+          onClick={this.handleClick.bind( this )}>
+          { this.props.children }
+        </Button>
+      )
+    } else {
+      return (
+        <div className="edit-details__button--active">
+          <Button styleName="button--save">Save changes</Button>
+          <Button styleName="button--cancel"
+            onClick={this.handleClick.bind( this )}>
+            Cancel
+          </Button>
+        </div>
+      )
+    }
   }
 }
+
+const mapDispatchToProps = dispatch => (
+  {
```

Comment:
Collapse the braces to the parens, please. Just looks super busy to me like this.